baracoda/operations.py

```diff
@@ -8,19 +8,30 @@
 from baracoda.formats import HeronFormatter
 from baracoda.orm.barcode import Barcode
 from baracoda.orm.barcodes_group import BarcodesGroup
+from baracoda.helpers import get_prefix_item
 
-logger = logging.getLogger(__name__)
+from flask import current_app
 
+logger = logging.getLogger(__name__)
 
 class BarcodeOperations:
-    def __init__(self, sequence_name: str, prefix: str):
+    def __init__(self, prefix: str):
         logger.debug("Instantiate....")
-        self.sequence_name = sequence_name
+        
         self.prefix = prefix
 
         self.__check_prefix()
 
-        self.formatter = HeronFormatter(prefix=self.prefix)
+        self.__set_prefix_item()
+
+        # if the prefix item does not exist the prefix is not valid
+        if not self.prefix_item:
+            raise InvalidPrefixError()
+        
+        # saves pulling it out of object every time
```

Comment:
Maybe adding a check that the keys are available in the dict would be safe? Someone could forget to add them in the config.

Reply:
Added check to config.py as per discussion.